libs/packages/sam-formly/src/lib/formly-reset/formly-reset.component.scss

```diff
@@ -0,0 +1,12 @@
+.usa-button--unstyled {
+  padding-right: 0;
+  width: initial;
+  padding-bottom: 0;
+  text-decoration: none;
+  &:hover, &:active {
+    background: transparent;
+    background: initial;
+    color: initial;
+    text-decoration: underline;
+  }
+}
```

Comment:
We don't want to give them the ability to change the appearance of the reset all.